src/assisted_installer_controller/reboots_notifier.go

```diff
@@ -20,8 +21,12 @@ import (
 const (
 	eventName            = "reboots_for_node"
 	eventMessageTemplate = "Node %s has been rebooted %d times before completing installation"
+	getNumRebootsRetries = 10
+	getNumRebootsTimeout = 6 * time.Minute
 )
 
+var getNumRebootsRetrySleepDuration = 15 * time.Second
```

Comment:
why keep it ourside of the const?

Reply:
I set it to short duration in unit tests